src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -116,7 +116,13 @@ export class MongoStorageAdapter {
     // encoded
     const encodedUri = formatUrl(parseUrl(this._uri));
 
-    this.connectionPromise = MongoClient.connect(encodedUri, this._mongoOptions).then(database => {
+    this.connectionPromise = MongoClient.connect(encodedUri, this._mongoOptions).then(client => {
+      // Starting mongoDB 3.0, the MongoClient.connect don't return a DB anymore but a client
+      // Fortunatelu, we can get back thr options
```

Comment:
There are a few typos in your comments. Also are you going to bump this to `3.0.1` for the index improvements?

Reply:
Yep after this one lands.